flare/gp.py

```diff
@@ -670,20 +673,24 @@ def from_dict(dictionary):
         new_gp.n_envs_prev = len(new_gp.training_data)
 
         # Save time by attempting to load in computed attributes
-        if len(new_gp.training_data) > 5000:
+        if dictionary.get('ky_mat_file'):
```

Comment:
maybe next time this part can be written as a function? it repeats again in the from_dict method.